fe/fe-core/src/main/java/com/starrocks/server/SharedDataStorageVolumeMgr.java

```diff
@@ -356,14 +352,35 @@ protected List<List<Long>> getBindingsOfBuiltinStorageVolume() {
         return bindings;
     }
 
-    private String[] getBucketAndPrefix() {
-        int index = Config.aws_s3_path.indexOf('/');
-        if (index < 0) {
-            return new String[] {Config.aws_s3_path, ""};
+    private static URI normalizeConfigPath(String uriStr, String defaultScheme, String configNameInErrMsg, boolean matchScheme)
+            throws InvalidConfException {
+        try {
+            URI uri = new URI(uriStr);
+            if (!uri.isAbsolute()) {
+                uri = new URI(defaultScheme + "://" + uriStr);
+            }
+            if (Strings.isNullOrEmpty(uri.getAuthority()) || uri.getPort() != -1) {
+                // no bucket or bucket name contains ':'
+                throw new InvalidConfException("");
```

Comment:
no message?

Reply:
intended, just fallback to the catch block, and rethrow a new InvalidConfException with detailed message.